eslintrc.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
@adamdoe why did you add this file?

Reply:
I had branched from test before the typescript changes made it in, and wanted to copy the same format.